backend/schema/message.py

```diff
@@ -11,22 +13,58 @@ class Reaction(BaseModel):
     character: str
 
 
-class Thread(BaseModel):
+class ThreadRequest(BaseModel):
     """
     Provides a base model for all threads
     """
-
-    sender_id: str
-    room_id: str
     text: str
     reactions: List[Reaction] = []
     files: List[AnyHttpUrl] = []
     saved_by: List[str] = []
-    is_pinned: bool = False
-    is_edited: bool = False
     created_at: str = str(datetime.utcnow())
 
 
+class Thread(ThreadRequest):
+    sender_id: str
+    room_id: str
+    org_id:str
+    message_id:str
+
+    @root_validator(pre=True)
+    @classmethod
+    def validates_channels(cls, values):
```

Comment:
Can this function be renamed to "validate_message"

Reply:
sure!